Exiled.API/Features/Player.cs

```diff
@@ -604,15 +604,7 @@ public bool IsBypassModeEnabled
         public bool IsMuted
         {
             get => ReferenceHub.dissonanceUserSetup.AdministrativelyMuted;
-            set
```

Comment:
Since we are making methods for this, and we still need to call IssuePersistentMute or RevokePersistentMute when this setter is called, we should do
`
set => value ? Mute() : UnMute();
`